vllm/attention/backends/flash_attn.py

```diff
@@ -710,6 +938,8 @@ def unified_flash_attention(
         # because different queries might have different lengths.
         assert decode_meta.max_decode_query_len is not None
         if decode_meta.max_decode_query_len > 1:
+            assert attn_type == AttentionType.DECODER, (
+                "Decoder only models support max_decode_query_len > 1")
```

Comment:
Change the comment to "Only decoder-only models support max_decode_query_len > 1"